core/src/main/java/edu/wpi/grip/core/operations/network/Vector2D.java

```diff
@@ -11,6 +11,7 @@
  * {@link Size}s are converted into this.
  */
 @Immutable
+@PublishableProxy({Point.class, Size.class})
```

Comment:
We _could_ iterate over all public constructors, but I like how this specifies exactly what's being proxied. It could also run into issues with publicly accessible helper constructors eg `Vector2D(double, double)`